sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/EndpointStateNotifierBase.java

```diff
@@ -56,7 +58,10 @@ public AmqpEndpointState getCurrentState() {
     void notifyError(Throwable error) {
         Objects.requireNonNull(error);
 
-        logger.error("Error occurred. {}", error.toString());
+        final String stackTrace = Stream.of(error.getStackTrace()).map(StackTraceElement::toString)
```

Comment:
I think we should consider updating ClientLogger to provide as much information about an exception (including stacktrace) as possible. For some reason, ClientLogger filters out exceptions before sending it to slf4j logger. If we don't do that, then slf4j will log the entire stacktrace.

Reply:
I'll create an issue for it and add a TODO in this to fix this logic when the issue is resolved.
https://github.com/Azure/azure-sdk-for-java/issues/5446